roles/marathon/defaults/main.yml

```diff
@@ -1,8 +1,30 @@
 ---
 # defaults file for marathon
-marathon_port: 8080
-consul_dir: /etc/consul.d
+marathon_enabled: true
+marathon_version: 'v0.8.2'
```

Comment:
I think we should use https://github.com/mesosphere/marathon/releases/tag/v0.9.0-RC2